tests/strict-equal.test.js

```diff
@@ -0,0 +1,30 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const strictEqual = require('../strict-equal.js')
+const assert = require('assert')
+
+describe('Comparison', () => {
+  describe('strictEqual', () => {
+    it('create a strictEqual function where 5 and "5" return false', () => {
+      // setup test data
+      var falseData = [5, '5']
+
+      // call the function with the test data
```

Comment:
The use of sending the array is a neat one. I think I have otherwise seen only where two params are being passed in such as `structEqual('5', 5)`